GetUser/handler.ts

```diff
@@ -193,6 +195,34 @@ export function GetUserHandler(
         );
         return { errorOrGroups, errorOrSubscriptions };
       })
+      .chain(taskResults =>
+        getGraphRbacManagementClient(adb2cCredentials)
+          .mapLeft(error =>
+            internalErrorHandler("Could not get the ADB2C client", error)
+          )
+          .chain(client =>
+            tryCatch(
+              () =>
+                client.users.list({
+                  filter: `signInNames/any(x:x/value eq '${email}')`
+                }),
+              error =>
+                internalErrorHandler(
+                  "Could not get user by email.",
+                  error as Error
+                )
+            ).map(userList => userList[0])
+          )
+          .map(adb2User => {
+            return {
+              ...taskResults,
+              token_name:
+                adb2User[
+                  `adb2User.extension_${adb2cCredentials.clientId}_token_name`
```

Comment:
what if the user does not have this attribute ? (may you try to delete ti into the ADB2C and call GetUser?)

Reply:
I think that many users does not have this attribute on ADB2C. In that case token_name should be null and it will remain null until we add token_name on Active Directory. Why should we delete the user if it does not have `token_name`?